.node-version

```diff
@@ -0,0 +1 @@
+20
```

Comment:
is this change needed for stripe?

Reply:
No not really. This project didn't build or run with any other node version initially so I just added this to make it easier to work with node versions. But now that I see, it builds fine with even the latest version so this might not be needed.